packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -836,42 +839,44 @@ export const RemoteServerFactory = {
                 lambdaContainerReused = true
                 app.sendMetric('LambdaCreated')
             }
-
-            // Proxy the request through to the server. When the response
-            // is done, context.succeed will be called with the response
-            // data.
-            awsServerlessExpress.proxy(
-                server,
-                event, // The incoming event
-                context, // The event context
-                'CALLBACK', // How the proxy signals completion
-                (err, response) => {
-                    // The 'response' parameter here is NOT the same response
-                    // object handled by ExpressJS code. The awsServerlessExpress
-                    // middleware works by sending an http.Request to the Express
-                    // server and parsing the HTTP response that it returns.
-                    // Wait util all pending metrics have been sent, and any pending
-                    // response caching to complete. We have to do this now, before
-                    // sending the response; there's no way to do it afterwards
-                    // because the Lambda container is frozen inside the callback.
-
-                    // We return this Promise, but the awsServerlessExpress object
-                    // doesn't make any use of it.
-                    return (
-                        app._requestMonitor
-                            ._waitForResponses()
-                            .then(() => app.metrics.flush())
-                            // Now call the Lambda callback to complete the response
-                            .then(() => callback(err, processLambdaResponse(response)))
-                        // DON'T add any then() handlers here, after the callback.
-                        // They won't be called after the response is sent, but they
-                        // *might* be called if the Lambda container running this code
-                        // is reused, which can lead to odd and unpredictable
-                        // behaviour.
-                    )
-                }
-            )
         }
+
+        const server = serverlessExpress({
```

Comment:
In the docs I see the `serverlessExpress` app returns the `handler` and not the `server` although I don't know how you get the `server`?

From the docs

```
exports.handler = serverlessExpress({ app })
```

Reply:
Ah, that's a good call out... it would explain why we're not seeing `server.close()`